packages/driver/src/cypress/error_utils.ts

```diff
@@ -549,9 +549,11 @@ const errorFromUncaughtEvent = (handlerType: HandlerType, event) => {
     errorFromProjectRejectionEvent(event)
 }
 
-const logError = (Cypress, handlerType: HandlerType, err, handled = false) => {
+const logError = (Cypress, handlerType: HandlerType, err: unknown, handled = false) => {
+  const error = toLoggableError(err)
+
   Cypress.log({
-    message: `${err.name}: ${err.message}`,
+    message: `${error.name}: ${error.message}`,
```

Comment:
```suggestion
    message: `${error.name || 'Error'}: ${error.message}`,
```

It's an unlikely edge case, but since we're only checking that the error has a message to be loggable (which I think is fine), this will keep `reject({ message: 'error message' })` from logging `undefined: error message`.

Reply:
We likely can can simplify this check to either use the Error or the value provided:

```suggestion
message: isLogabbleError(err) ? `${err.name || 'Error'}: ${err.message}` : err
```

Reply:
@emilyrohrbough I think your suggestion makes things more complicated, it would also not pass the `err name: error cause` format to Cypress.log